components/MarkdownTextBlock/src/TextElements/MyFlowDocument.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using HtmlAgilityPack;
+using Markdig.Syntax;
+
+namespace CommunityToolkit.Labs.WinUI.MarkdownTextBlock.TextElements;
+
+public class MyFlowDocument : IAddChild
+{
+    private HtmlNode? _htmlNode;
+    private RichTextBlock _richTextBlock = new RichTextBlock();
+    private MarkdownObject? _markdownObject;
+
+    // useless property
+    public TextElement TextElement { get; set; } = new Run();
+    //
+
+    public RichTextBlock RichTextBlock
+    {
+        get => _richTextBlock;
+        set => _richTextBlock = value;
+    }
+
+    public bool IsHtml => _htmlNode != null;
+
+    public MyFlowDocument()
+    {
+    }
+
+    public MyFlowDocument(MarkdownObject markdownObject)
+    {
+        _markdownObject = markdownObject;
+    }
+
+    public MyFlowDocument(HtmlNode node)
+    {
+        _htmlNode = node;
+    }
+
+    public void AddChild(IAddChild child)
+    {
+        TextElement element = child.TextElement;
+        if (element != null)
+        {
+#if !WINAPPSDK
+            if (element is Windows.UI.Xaml.Documents.Block block)
+#else
+            if (element is Microsoft.UI.Xaml.Documents.Block block)
+#endif
```

Comment:
You can define a using alias at the top for these types and use the base type here without the split. i.e. keep all the splits above like:

```cs
#if !WINAPPSDK
using Block = Windows.UI.Xaml.Documents.Block;
#else
using Block = Microsoft.UI.Xaml.Documents.Block;
#endif
```

Or even in the csproj like you did for the other global usings.

Reply:
Addressed